### DISCUSSION TITLE
How to stop Parcel format my CSS code automatically?

### DISCUSSION THREAD

**Discussion by zhujin917** [parcel-bundler/parcel#9796] [Q&A]
Firstly, I don't want Parcel to generate code like var t in my JS files. So I set optimize: false in package.json. Then I found my CSS code didn't be minified. So I tried using Transformer plugin with clean-css from NPM to minify my code. But sadly I found my CSS still be formatted after executing the setCode function.

Help! I will be appreciated if you can solve my problem.

**Comment by mischnic** [comment]
Add a `.parcelrc` file with 
```js
{
	"extends": "@parcel/config-default",
	"optimizers": {
		"*.js": []
	}
}
```

Which removes all JS optimizer plugins but leaves all other file types as they are.

**Reply by zhujin917** [reply]
Thanks. I will test it next weekend because I can't get my PC during school days.
